.github/workflows/build-test.yaml

```diff
@@ -36,6 +37,15 @@ jobs:
   build-kots:
     runs-on: ubuntu-18.04
     needs: [build-web]
+
+    # This workflow trigger may lead to malicious PR authors being able to obtain repository write permissions or stealing repository secrets. 
+    # Please read https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
+    # only run this workflow if:
+    #   not a fork or user is dependabot or PR is labeled with '@actions/safe-to-test'
+    if: >
```

Comment:
Why is this the only job that has this condition?

Reply:
@divolgin added the ci-gate workflow for the rest of the jobs.